site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,11 +16,60 @@ updated: 2022-02-17
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 102: New manifest field "optional_host_permissions" {: #m102-optional-host-permissions }
+
+April 4, 2022
+
+Added a new manifest key named `optional_host_permissions` for Manifest V3. This behaves the exact
+same as specifying hosts in `optional_permissions` in Manifest V2.
+
+### Chrome 102: injectImmediately property in scripting.executeScript() {: #m102-injectimmediately }
+
+April 4, 2022
+
+`chrome.scripting.executeScript()` now accepts an optional `injectImmediately` property.  If present
+and set to true, the script will inject into the target as soon as possible, rather than waiting for
+`document_idle`. Note that this is not a guarantee the script will inject before the page is loaded
+since the page continues to load while the API call is being made.
+
+### Chrome 102: Omnibox API support in Manifest V3 {: #m102-omnibox }
+
+March 31, 2022
+
+The [Omnibox API](/docs/extensions/reference/omnibox) can now be used in service worker based
+extensions. Previously, some of this API's methods would throw on invocation due to internal
+dependencies on DOM capabilities.
+
+### Chrome 102: wasm-unsafe-eval allowed in Manifest V3 CSP {: #m102-wasm }
+
+March 22, 2022
+
+Manifest V3 extensions can now include `wasm-unsafe-eval` in their `content_security_policy` declarations. This change allows Manifest V3 extensions to use Wasm.
+
 ### Docs update: Chrome Web Store item discovery {: #cws-discoevery-doc }
 
+March 21, 2022
+
 [Discovery on Chrome Web Store](/docs/webstore/discovery/) gives an overview of how users find items
 on the Chrome Web Store and how our editors select items to feature.
 
+### Chrome 101: Improved declarativeNetRequest domain conditions {: #m101-dnr-conditions }
+
+March 9, 2022
+
+[declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest/) rule conditions have been
+updated to allow extensions to better target requests based on the request's "request" and
+"initiator" domains. The relevant condition properties are `initiatorDomains`,
+`excludedInitiatorDomains`, `requestDomains`, and `excludedRequestDomains`. Also, the engineer that
+implemented this change announced this change and answered some questions form developers in this
+[chromium-extensions
+thread](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/4971ZS9cI7E). 
+
+### Chrome 100: Resolved issue with scripting.executeScript on newly created tabs {: #m100-executescript-bugfix }
+
+There was a longstanding issue where calling `scripting.executeScript()` on a newly created tab or
+window could fail. These should be resolved now and it should execute as you would expect.
```

Comment:
Shorter:

```suggestion
Fixed a longstanding issue where calling `scripting.executeScript()` on a newly created tab or
window could fail.
```

Saying "we fixed it and it should now work as you expect" seems redundant (since I'd assume "fixing it" requires "making it work as I'd expect")